typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -248,31 +261,55 @@ export abstract class HyperlaneAppGovernor<
           config: accountConfig,
         });
         if (!callRemote.to || !callRemote.data) {
-          return SubmissionType.MANUAL;
+          return {
+            type: SubmissionType.MANUAL,
+            chain,
+            call,
+          };
         }
         const encodedCall: AnnotatedCallData = {
           to: callRemote.to,
           data: callRemote.data,
           value: callRemote.value,
           description: `${call.description} - interchain account call from ${origin} to ${chain}`,
         };
-        const subType = await this.inferCallSubmissionType(origin, encodedCall);
+        const { type: subType } = await this.inferCallSubmissionType(
+          origin,
+          encodedCall,
+          async (chain: ChainName, submitterAddress: Address) => {
```

Comment:
Does this need to be async?

Reply:
nice catch, this was a relic of a previous iteration where it did some async stuff, ill rm